spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'guests', type: :feature do
+
+  describe 'GET #show' do
+
+    before(:each) do
+      @hotel = Hotel.create(name: 'Green Mountian Inn', location: 'Stowe')
+      @room1 = @hotel.rooms.create(rate: 200, suite: 'Super')
+      @room2 = @hotel.rooms.create(rate: 150, suite: 'Deluxe')
+      @guest1 = Guest.create(name: 'Kye', nights: 6)
+      @guests_room = GuestsRoom.create(guest: @guest1, room: @room1)
+    end
+
+    it "shows guests with information on show page" do
+      visit "/guests/#{@guest1.id}"
+
+      expect(page).to have_content(@guest1.name)
+      expect(page).to have_content("Hotel: #{@hotel.name}")
+      expect(page).to have_content("Suite: #{@room1.suite}")
+      expect(page).to have_content("Nightly Rate: $#{@room1.rate}")
```

Comment:
Ideally when we're testing anything with iteration (in this case, rooms), we'd like to see multiple items displayed in the test. So it would be good to have another GuestRoom record for the 2nd room and then test that room info for both rooms is present. `within` blocks could help you verify that info is in the correct place, as well.